_posts/2017-12-30-JS-Conf-Budapest-day-2.md

```diff
@@ -0,0 +1,387 @@
+</blockquote>
+
+#### But what do developers think of code reviews?
+
+To know this, Vaidehi did a [survey on Twitter](https://twitter.com/vaidehijoshi/status/835597499813494788) and got about 500 responses.
+The survey had questions with a scale of 1 to 10, where 1 was strongly disagree and 10 was strongly agree.
+These are the stats:
+
+##### The quantitive data
+
+The question "Code reviews are beneficial to my team" had a clear answer.
+The average score was around 9 for most languages, with the top 3 containing Swift at an average of 9.46, Ruby at an average of 9.19 and JavaScript at an average of 9.1.
+<p class="image"><img src="/img/js-conf-budapest-2017/vaidehi-joshi-beneficial.jpeg" alt="Code reviews are beneficial to my team" width="100%"></p>
+
+Another question was "How many pull requests are reviewed", on which the majority answered that all pull request were reviewed.
+However, about 10% of the answers indicated that pull requests where only reviewed when someone was explicitly requested to review.
+<p class="image"><img src="/img/js-conf-budapest-2017/vaidehi-joshi-how-many-pull-request-reviews.jpeg" alt="How many pull requests are reviewed" width="100%"></p>
+
+##### The qualitative data
+
+So, most developers think code reviews are needed and state that all code is being reviewed.
+But what do they think of the quality of code reviews?
+
+<blockquote class="clear"><p>
+Ultimately, what seemed to make or break a code review experience depended upon two things: how much energy was spent during the review process and how much substance the review itself had.
+</p></blockquote>
+
+A code review was bad (and left a bad taste in the reviewer’s and reviewee’s mouth) if there wasn’t enough energy spent on the review, or if it lacked substance.
+On the other hand, if a code review process was thorough and time was spent reviewing aspects of the code in a substantive way, it left a much more positive impression overall on both the reviewer and the reviewee.
+
+###### ENERGY
+
+On the question "Who all is doing the review and how much time are they spending on it?", a lot of things could be learned.
+
+* A developer blindly thumbs-up everything or the second or third reviewer is more likely to agree when already seeing an approval.
+This makes the code review a formality, which doesn't carry any weight.
+* A review is performed different depending on who submits.
+Seniors get no feedback, while juniors are picked to death.
+The reviews are unfair and can break confidence.
+* Commits are too big, which cause long review time, which in turn has a bad effect on future branches/PRs/merges.
+Long review times take too much energy, which causes them to be postponed.
+
+###### SUBSTANCE
+
+The question "What exactly is someone saying, doing, or making another person feel while they review their code?" brought these answers.
+
+* A reviewee taking all feedback at face value, having a mentality of "see red squiggle, fix red squiggle".
```

Comment:
Maybe better this way: "An assessor who takes all the feedback for his own account, having ..."